app/assets/stylesheets/application.css.scss

```diff
@@ -15,64 +15,69 @@
  *= require foundation_and_overrides
 
  */
+ body, h2, h4 {
+   background-color: white;
+   font-family: 'Karla', sans-serif;
+ }
 
-body {
-  background-color: white;
-}
+ .review_body {
+   max-width: 500px;
+   word-wrap: break-word;
+ }
 
-.name {
-  padding-top: 7px;
-  padding-right: 10px;
-}
+ .top-bar {
+   background-color: #FCD720;
```

Comment:
Color literals like `#FCD720` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#FCD720` should be written as `#fcd720`